src/tests/const.test.js

```diff
@@ -0,0 +1,78 @@
+import { createHeadlessForm } from '../createHeadlessForm';
+
+it('Should work for number', () => {
```

Comment:
question: Is this the moment we start to break our tests into smaller files? If so this should be a `describe('validations: const', ())` with all the `it` inside it.

Reply:
yes it is, no more massive test files 😅 , will fix this.